Blog: Best LaunchDarkly alternatives (#7213)
PostHog/posthog.com: contents/blog/best-launchdarkly-alternatives.mdx

### Key features

- 🚩 **Feature flags:** Safely rollout features with [local evaluation](/docs/feature-flags/local-evaluation) (for faster performance), JSON payloads, and instant rollbacks.

- 🧪 **A/B tests:** Up to 9 test variations, primary and secondary metrics. Automatically calculate test duration, sample size, and statistical significance.

- 📈 **Product analytics:** Funnels, user paths, retention analysis, custom trends, and dynamic user cohorts. Also supports [SQL insights](/docs/product-analytics/sql) for power users.

- 💬 **Surveys:** Target surveys by event or user properties. Templates for [net promoter score (NPS)](/templates/nps-survey), [product-market fit (PMF)](/templates/pmf-survey) surveys, and more.

- 📺 **Session replays:** Including event timelines, console logs, network activity, and 90-day data retention.

### How does PostHog compare to LaunchDarkly?

The core features for feature flags and experimentation are similar between the two. The big difference is that PostHog is free, open-source, and self-serve, while LaunchDarkly has automations.

> Also, for LaunchDarkly, features like integrations, API controls, and reusable segments are only available on their Pro plan. Much of their automation and governance functionality is only available on their Enterprise plans.

Reviewer: ```suggestion
Also, for LaunchDarkly, features like integrations, API controls, and reusable segments are only available on its Pro plan. Much of its automation and governance functionality is only available on LaunchDarkly's Enterprise plans.
```

3. **They need a complete picture of users:** PostHog includes every tool necessary to understand users and improve products. This means creating funnels to track conversion, watching replays to see where users get stuck, testing solutions with A/B tests, and gathering feedback with user surveys.

> #### Bottom line
> Being free, self-serve, and sharing many of the same features, PostHog is a great alternative to LaunchDarkly. This is especially true for startups and scale-ups looking for all the product and data tools they need in one.

<ArrayCTA />

<br />

## 2. Statsig

- **Founded:** 2021
- **Similar to:** DevCycle, PostHog
- **Typical users:** Engineering and DevOps teams
- **Typical customers:** Engineering-focused B2B companies

![Statsig](../images/blog/best-launchdarkly-alternatives/statsig.png)

### What is Statsig?

Statsig provides tools like feature flags, experimentation, and analytics to help companies build better products. Teams use Statsig to take the risk out of releases, experiment with new features, and monitor changes.

> It also includes a warehouse-native mode to connect directly and utilize your current data warehouse like Snowflake.

Reviewer: ```suggestion
It also includes a warehouse-native mode to connect directly and utilize your data warehouse, such as Snowflake.
```

Reviewer: "Like" would imply it works in a similar way to Snowflake.

  <ComparisonRow column1={true} column2={true} feature="Major SDKs" description="Pre-built SDKs for JavaScript, Python, iOS, Android, Go, and more" />
  <ComparisonRow column1={true} column2={true} feature="API" description="Evaluate and edit flags and A/B tests with an API" />
  <ComparisonRow column1={true} column2={true} feature="Automations" description="Trigger flag states based on schedules, triggers" />
  <ComparisonRow column1={true} column2={true} feature="Imports and exports" description="Import and export data to and from other popular tools" />
  <ComparisonRow column1={true} column2={true} feature="Governance" description="Audit log, roles-based access, permissions" />
  <ComparisonRow column1={"SDKs"} column2={"SDKs"} feature="Open source" description="Audit code, contribute to roadmap, and build integrations" />
</ComparisonTable>

### How popular is Statsig?

According to [BuiltWith](https://trends.builtwith.com/analytics/Statsig), 564 of the top million sites use Statsig. This is just under LaunchDarkly's 876.

### Why do companies use Statsig?

According to G2, users are big fans of Statsig because:

> 1. **Experiments-focused:** Statsig provides all the tools to run successful experiments. Reviewers write this enables them to ship faster and create an experimentation mindset.
> 2. **Responsiveness:** The Statsig team is responsive to user issues and concerns. Reviewers appreciate how helpful support is. 
> 3. **Documentation:** Thanks to the solid documentation of SDKs and features, in combination with a simple UX, reviewers find Statsig easy to set up and use.

Reviewer: ```suggestion
1. **Experiments-focused:** Statsig provides all the tools to run successful experiments. Reviewers write this enables them to ship faster and create an experimentation mindset.

2. **Responsiveness:** The Statsig team is responsive to user issues and concerns. Reviewers appreciate how helpful support is. 

3. **Documentation:** Thanks to the solid documentation of SDKs and features, in combination with a simple UX, reviewers find Statsig easy to set up and use.
```

  <ComparisonRow column1={true} column2={true} feature="Funnel tests" description="Use conversion funnels as goals for your A/B test" />
  <ComparisonRow column1={false} column2={false} feature="Advanced analytics" description="Custom SQL, paths, and retention analysis with flag data and beyond." />
  <ComparisonRow column1={true} column2={true} feature="Major SDKs" description="Pre-built SDKs for JavaScript, Python, iOS, Android, Go, and more" />
  <ComparisonRow column1={"Edit only"} column2={true} feature="API" description="Evaluate and edit flags and A/B tests with an API" />
  <ComparisonRow column1={false} column2={true} feature="Automations" description="Trigger flag states based on schedules, triggers" />
  <ComparisonRow column1={true} column2={true} feature="Imports and exports" description="Import and export data to and from other popular tools" />
  <ComparisonRow column1={true} column2={true} feature="Governance" description="Audit log, roles-based access, permissions" />
  <ComparisonRow column1={"SDKs"} column2={"SDKs"} feature="Open source" description="Audit code, contribute to roadmap, and build integrations" />
</ComparisonTable>

### How popular is Split?

> According to [BuiltWith](https://trends.builtwith.com/analytics/Split), 236 of the top million sites use Split as of November 2023. This is roughly 1/3 of LaunchDarkly's 876.

Reviewer: ```suggestion
According to [BuiltWith](https://trends.builtwith.com/analytics/Split), 236 of the top million sites use Split as of November 2023. This is roughly a third of LaunchDarkly's 876.
```

### Why do companies use Split?

According to G2, reviewers are big fans of Split because:

> 1. **Ease of setup:** Reviewers appreciate how easy Split is to set up thanks to their SDKs, guides, and user interface.
> 2. **More confident releases:** The tools Split provides, like easy feature kill switches, alerts, and monitoring enable reviewers to be confident in their feature releases.
> 3. **Intuitive interface:** Creating a new flag is simple, but Split maintains the depth more advanced users require as well. Reports are easy to understand.

Reviewer: ```suggestion
1. **Ease of setup:** Reviewers appreciate how easy Split is to set up thanks to their SDKs, guides, and user interface.

2. **More confident releases:** The tools Split provides, like easy feature kill switches, alerts, and monitoring enable reviewers to be confident in their feature releases.

3. **Intuitive interface:** Creating a new flag is simple, but Split maintains the depth more advanced users require as well. Reports are easy to understand.
```

  <ComparisonRow column1={false} column2={false} feature="Advanced analytics" description="Custom SQL, paths, and retention analysis with flag data and beyond." />
  <ComparisonRow column1={true} column2={true} feature="Major SDKs" description="Pre-built SDKs for JavaScript, Python, iOS, Android, Go, and more" />
  <ComparisonRow column1={"Edit only"} column2={true} feature="API" description="Evaluate and edit flags and A/B tests with an API" />
  <ComparisonRow column1={true} column2={true} feature="Automations" description="Trigger flag states based on schedules, triggers" />
  <ComparisonRow column1={"Exports only"} column2={true} feature="Imports and exports" description="Import and export data to and from other popular tools" />
  <ComparisonRow column1={"No audit log"} column2={true} feature="Governance" description="Audit log, roles-based access, permissions" />
  <ComparisonRow column1={"SDKs"} column2={"SDKs"} feature="Open source" description="Audit code, contribute to roadmap, and build integrations" />
</ComparisonTable>

### How popular is DevCycle?

> BuiltWith doesn't have data on DevCycle's usage, but we can use organic traffic as a proxy. Ahrefs estimates its organic traffic to be 3.5k per month, while it estimates LaunchDarkly at 46.3k per month, nearly 15x more popular.

Reviewer: I think Google trends might be a better alternative for this: https://trends.google.com/trends/explore?date=today%205-y&q=devcycle,launchdarkly,posthog

Come to think of it, maybe we should add this going forward?

In any case, Ahrefs measures traffic to any content, whereas as Trends will show searches for their brand name in Google.

Author: BuiltWith feels more tangible, so I think we should try to use it if possible. For some (like LaunchDarkly), it might be better to use Trends because they don't get captured well by however BuiltWith measures tools.
